sdk/node/test/unit/chain-tests.js

```diff
@@ -1,20 +1,24 @@
-/*
-Copyright IBM Corp. 2016 All Rights Reserved.
+/**
+ * Copyright 2016 IBM
```

Comment:
Please align the copyright with the template [here](https://github.com/hyperledger/fabric/blob/master/docs/dev-setup/headers.txt).